node/server.go

```diff
@@ -342,6 +343,16 @@ func Run(ctx context.Context, opts Options) error {
 		node: nd,
 	}
 
+	if opts.UseInflux {
```

Comment:
@tchardin this is where the initial check-in happens -- we just need to run a similar operation whenever a retrieval occurs and calculate stats on that retrieval accordingly